### PR TITLE
content/quickstart: fix Java quickstarts and screenshots

### LINKED ISSUE #243: content/quickstart: Java samples are sometimes wrong, don't compile and don't have the right screenshots
This was reported offline Mary Tabasko who tried using the Java quickstart for metrics and found many bunch of problems of which I'll list a few:
a) We don't mention in requirements that the user needs to install Maven, and how to do it
b) `M_LATENCY_MS` is redefined, hence most of the quickstart doesn't compile -- this was a bug introduced by https://github.com/census-instrumentation/opencensus-website/commit/753ed265da5f81f199341a9b0ac1e0204b867a36 aka PR #206
c) In the snippets for creating views, the Views constructor Name.create is given the wrong number of arguments i.e.
```diff
--- before	2018-08-18 18:05:37.000000000 -0700
+++ after	2018-08-18 18:05:48.000000000 -0700
@@ -1,10 +1,6 @@
 View[] views = new View[]{
-        View.create(Name.create(
-            "demo/latency", "The distribution of latencies", M_LATENCY_MS, latencyDistribution, Collections.singletonList(KEY_METHOD))),
-        View.create(Name.create(
-            "demo/lines_in", "The number of lines read in from standard input", M_LATENCY_MS, countAggregation, noKeys)),
-        View.create(Name.create(
-            "demo/errors", "The number of errors encountered", M_ERRORS, countAggregation, noKeys)),
-        View.create(Name.create(
-            "demo/line_length", "The distribution of line lengths", M_LINE_LENGTHS, lengthsDistribution, noKeys))
+        View.create(Name.create("demo/latency"), "The distribution of latencies", M_LATENCY_MS, latencyDistribution, Collections.singletonList(KEY_METHOD)),
+        View.create(Name.create("demo/lines_in"), "The number of lines read in from standard input", M_LINES_IN, countAggregation, noKeys),
+        View.create(Name.create("demo/errors"), "The number of errors encountered", M_ERRORS, countAggregation, noKeys),
+        View.create(Name.create("demo/line_length"), "The distribution of line lengths", M_LINE_LENGTHS, lengthsDistribution, noKeys)
     };
```
which obviously means that those snippets don't compile
d) Wrong Stackdriver screenshots. The screenshots for OpenCensus-Go were accidentally used, yet we should have used the proper ones. A copy and paste problem perhaps from re-using the Go template. We must be able to generate fresh screenshots

### PR DESCRIPTION
**Pull request by odeke-em**
Fixes #243

Addresses some problems reported as it:
* Ensures that for both tracing and metrics that we mention
that `Apache Maven` is needed lest the user gets surprised
later on when we ask them to run `mvn` commands
* Fix erraneous replacements of `M_LINES_IN` by `M_LATENCY_MS`
that accidentally crept in from PR #206
* Fix bad code compilation problems from `View.create(Name.create`
being given the wrong arguments, hence some snippets couldn't
compile
* Uses freshly generated Stackdriver Tracing and Metrics screenshots
as Go screenshots were accidentally used

### CODE REVIEW

File: content/quickstart/java/tracing.md [hunk 1]
```diff
@@ -1040,7 +1044,7 @@ public class Repl {
         }
     }
 
-    private static void readEvaluateProcess(BufferedReader in) throws IOException {
+    private static void repl(BufferedReader in) throws IOException {
```
Comment (bogdandrutu) [inline content/quickstart/java/tracing.md]: Java likes long names for methods in general. Better to use readEvaluateProcessLine?

Comment (odeke-em) [reply content/quickstart/java/tracing.md]: Aye aye, I actually thought of doing that but shortened it for Go and Python :) Let me update that in a second, thanks!

**Comment by odeke-em** [comment]
@bogdandrutu I've updated `s/repl/readEvaluateProcessLine/g` PTAL.

**Comment by odeke-em** [comment]
Thank you for the review @bogdandrutu!

**Comment by songy23** [comment]
Great, thanks for the fix!

**Pull request merged**